azure/v11.4.0/README.md

```diff
@@ -0,0 +1,60 @@
+## :zap:  Giant Swarm Release 11.4.0 for Azure :zap:
+
+**If you are upgrading from 11.3.3, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+In this release NGINX Ingress Controller LoadBalancer Service external traffic policy has been made configurable, and the policy default has been changed from `Cluster` to `Local`. These changes:
+
+- Enable source IP preservation, needed for IP based security access control.
+- Remove unnecessary ingress traffic intra-cluster network hops, increasing concurrent connection capacity and reducing latency.
+- Allow configuration to be reverted, where these defaults and associated tradeoffs are found not to be appropriate.
+
+Management of NGINX IC LoadBalancer Service is moved from azure-operator to NGINX IC App itself to:
+
+- Enable external traffic policy configurability, in a way consistent with other NGINX IC configuration options.
+- Lay the foundation for making NGINX IC App optional and not pre-installed in a future Azure platform release.
+
+When upgrading existing clusters from older Azure platform releases, migration of LoadBalancer Service is automated but not fully automatic. Therefore we ask of you to delegate cluster upgrades to your SE.
+
+**Note for SEs:** After cluster upgrade to 11.4.0, both old `ingress-loadbalancer` LoadBalancer Service managed by azure-operator and new one `nginx-ingress-controller` managed by NGINX IC App remain on the cluster. To switch the ingress traffic to the new LoadBalancer and remove old NGINX LoadBalancer Service without downtime please:
+
+- Together with the customer have any firewall in front of NGINX reconfigured to allow both old and new LoadBalancer Service IPs.
+- Next use [the migration script](https://github.com/giantswarm/azure-operator/blob/master/scripts/migrate-nginx-ingress-controller.sh) to switch DNS records to the new load balancer IP. The script ensures IP is assigned to the new LB, and also that the cluster DNS records resolve to it instead of old IP.
+- Now delete the old NGINX IC LoadBalancer Service.
+- Finally, ensure that the tenant cluster's
+  - Azure `kubernetes` load balancer backend pool includes the worker VM scale set, and that
+  - All the worker VM scale set instances have latest model applied.
+
+**Note when upgrading from v11.2.x to v11.3.x or v11.4.x:**
+
+This release contains the replacement of CoreOS with Flatcar introduced in v11.3.0. Please carefully read release notes for 11.3.0 including Flatcar OS migration [steps](https://github.com/giantswarm/releases/tree/master/azure/v11.3.0).
+
+**Note for future v11.4.x releases:**
+
+Please persist this and the above note until all customers are in v11.3.0 and above.
+
+Below, you can find more details on components that were changed with this release.
+
+### azure-operator [v4.1.0](https://github.com/giantswarm/azure-operator/releases/tag/v4.1.0)
```

Comment:
This should probably link to the `CHANGELOG` entry for v4.1.0. It's not there atm, but that needs to be fixed.